packages/flutter_tools/lib/src/android/android_sdk.dart

```diff
@@ -262,7 +268,7 @@ class AndroidSdk {
     return null;
   }
 
-  String? getCmdlineToolsPath(String binaryName) {
+  String? getCmdlineToolsPath(String binaryName, {bool skipTools = false}) {
```

Comment:
Is there a more specific name than `skipTools` for this flag? Maybe `skipOldTools` (supposing that I'm reading the comments in the function correctly)?

Reply:
Also does the `skipTools = true` case have test coverage?

Reply:
done!